src/Compilers/Core/CodeAnalysisTest/InternalUtilities/EnumerableExtensionsTests.cs

```diff
@@ -1,17 +1,19 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System;
+using System.Collections.Generic;
 using Xunit;
 
 namespace Roslyn.Utilities.UnitTests.InternalUtilities
 {
+    using System.Linq;
```

Comment:
Please move the using outside the namespace.

Reply:
Will fix. Somehow the "Add using" added it here :(

Reply:
Actually, this `using` is required inside the namespace. Both `Roslyn.Utilities` and `System.Linq` namespaces have `EnumerableExtensions`. We want to test the latter here.